hfgl/cli.py

```diff
@@ -138,6 +138,7 @@ def synthesize(
     device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
     checkpoint = torch.load(generator_path, map_location=device)
     data = torch.load(data_path, map_location=device)
+    # TODO: Should we also wrap this load_hifigan_from_checkpoint in case the checkpoint is not a Vocoder?
```

Comment:
To me the answer depends on this question: if the user calls `hfgl synthesize` with the wrong type of model, do they get a stack dump here? If so, the wrapper makes sense.